### PR TITLE
blog: march commit of the month

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

File: blog/_posts/2020-04-02-march-commit-of-the-month.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+![A photo of the Tilt UI with green and red icons next to each resource](/assets/images//march-2020-commit-of-the-month/colorblind1.png)
+
+Can you tell at a glance which of these resources are up, and which ones have errored? It’s hard. If you were red-green colorblind it would be even harder. It might look something like this:
+
+![A photo of the Tilt UI with green and red icons next to each resource modified to show what it might look like to a colorblind person](/assets/images/march-2020-commit-of-the-month/colorblind1-mod.png)
+
+While we expect most people to be using the Web UI (which is more fully featured and happened to not exhibit this problem by dint of a different design) if your workflow lives in the terminal we want to support that too. So in [842b83131f78431dd1ba8522e6288226a09c4cfc](https://github.com/windmilleng/tilt/commit/842b83131f78431dd1ba8522e6288226a09c4cfc) we improved the terminal UI’s accessibility by adding status icons that are structurally distinct from one another in addition to having distinct hues. Now Tilt’s status icons look like this:
```
Comment (nicks) [inline blog/_posts/2020-04-02-march-commit-of-the-month.md:28]: this text isn't quite accurate. The WebUI had this problem when the user reported it. We fixed it in a different way in a different PR https://github.com/windmilleng/tilt/pull/2955/

Comment (jazzdan) [reply blog/_posts/2020-04-02-march-commit-of-the-month.md]: Ah I see, thanks! I looked for something like that but missed it the first time. Should be fixed now.

**Comment by jazzdan** [comment]
@nicks ping!

File: blog/_posts/2020-04-02-march-commit-of-the-month.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - cotm
+  - accessibility
+keywords:
+  - tilt
+  - accessibility
+---
+
+Details matter. Especially when those details impact usability. That’s why we paid close attention when we got reports that Tilt’s terminal UI was not colorblind friendly. Take a look at this screenshot:
+
+![A photo of the Tilt UI with green and red icons next to each resource](/assets/images//march-2020-commit-of-the-month/colorblind1.png)
```
Comment (nicks) [inline blog/_posts/2020-04-02-march-commit-of-the-month.md:22]: remove the double slash?

File: blog/_posts/2020-04-02-march-commit-of-the-month.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Can you tell at a glance which of these resources are up, and which ones have errored? It’s hard. If you were red-green colorblind it would be even harder. It might look something like this:
```
Comment (nicks) [inline blog/_posts/2020-04-02-march-commit-of-the-month.md:24]: Love this graf

File: blog/_posts/2020-04-02-march-commit-of-the-month.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+![A photo of the Tilt UI with green and red icons next to each resource modified to show what it might look like to a colorblind person](/assets/images/march-2020-commit-of-the-month/colorblind1-mod.png)
+
+While we expect most people to be using the Web UI (which is more fully featured though it exhibited a similar problem that was [fixed in a different commit](https://github.com/windmilleng/tilt/commit/0c1985c3e2ddeeedce7fb5633753d23730887f62)) if your workflow lives in the terminal we want to support that too. So in [842b83131f78431dd1ba8522e6288226a09c4cfc](https://github.com/windmilleng/tilt/commit/842b83131f78431dd1ba8522e6288226a09c4cfc) we improved the terminal UI’s accessibility by adding status icons that are structurally distinct from one another in addition to having distinct hues. Now Tilt’s status icons look like this:
```
Comment (maiamcc) [inline blog/_posts/2020-04-02-march-commit-of-the-month.md:28]: This paragraph reads awkwardly to me, here's an attempt at wordsmithing, take or leave as you like:

While we expect most of our users to be using the Web UI (which is more fully featured than the terminal UI[1]), if your workflow lives in the terminal, we want to support that too.
1: the Web UI at one point had a similar accessibility problem, which we fixed in a different commit (link)

File: blog/_posts/2020-04-02-march-commit-of-the-month.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+![A photo of the Tilt new UI with green and red icons of different shapes for each different state](/assets/images/march-2020-commit-of-the-month/now.png)
+
+Even in grayscale it’s now easy to tell that these icons represent different states.
```
Comment (maiamcc) [inline blog/_posts/2020-04-02-march-commit-of-the-month.md:32]:
```suggestion
Even in grayscale, it’s now easy to tell that these icons represent different states.
```

**Pull request merged**